apps/solver/base.hu.i18n

```diff
@@ -1,29 +1,29 @@
 SolverApp = "Egyenletek"
-SolverAppCapital = "Egyenletek"
-AddEquation = "Egyenlet hozzáadása"
-ResolveEquation = "Oldja meg az egyenletet"
-ResolveSystem = "A rendszer megoldása"
+SolverAppCapital = "EGYENLETEK"
+AddEquation = "Egyenlet hozzáadás"
+ResolveEquation = "Egyenletet megoldása"
+ResolveSystem = "Rendszer megoldása"
 UseEquationModel = "Használjon egyenleti sablont"
-RequireEquation = "A bemenetnek egyenletnek kell lennie"
-UnrealEquation = "Unreal egyenlet"
-UndefinedEquation = "Definiálatlan egyenlet"
-TooManyVariables = "Túl sok ismeretlen"
+RequireEquation = "A bemenetnek csak egyenlet lehet"
+UnrealEquation = "Irreális egyenlet"
+UndefinedEquation = "Egy egyenlet meghatározatlan "
+TooManyVariables = "Túl sok ismeretlen van"
 NonLinearSystem = "A rendszer nem lineáris"
 Solution = "Megoldás"
 ApproximateSolution = "Hozzávetöleges megoldás"
 SearchInverval = "Keresési intervallum"
 NoSolutionSystem = "A rendszernek nincs megoldása"
 NoSolutionEquation = "Az egyenletnek nincs megoldása"
-NoSolutionInterval = "Nem találtunk megoldást az intervallumban"
-EnterEquation = "Írja be az egyenletet"
-InfiniteNumberOfSolutions = "Végtelen számú megoldás létezik"
-ApproximateSolutionIntervalInstruction0 = "Adja meg a keresés intervallumát"
-ApproximateSolutionIntervalInstruction1 = "egy hozzávetöleges megoldáshoz"
-OnlyFirstSolutionsDisplayed0 = "Csak az elsö 10 megoldás"
-OnlyFirstSolutionsDisplayed1 = "megjelenik"
+NoSolutionInterval = "Nincs megoldás ebben az intervallumban"
+EnterEquation = "Írjon be egy egyenletet"
+InfiniteNumberOfSolutions = "Végtelen menyi megoldások léteznek"

```

Comment:
Once again the other one was good, the changed one makes no sense grammatically. We should use " Végtelen számú megoldás létezik"